-1

Tested at: 73690b498c3cf7710f3c6ebc126eaa6ffac04e0f

You can see the results of the tests here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17252/17387/summary.html

I found follow errors while testing this PR

**Failed tests:  Build**

* **Build**:

I found an error when building: 
 
 <pre>>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2017-01-23-1100/src/SimMuon/RPCDigitizer/test/RPCDigiReader.cc 
>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2017-01-23-1100/src/SimMuon/RPCDigitizer/test/RPCFakeEvent.cc 
>> Compiling edm plugin /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_9_0_X_2017-01-23-1100/src/SimMuon/RPCDigitizer/test/RPCEventDump.cc 
>> Building edm plugin tmp/slc6_amd64_gcc530/src/SimMuon/RPCDigitizer/test/RPCEventDump/libRPCEventDump.so
/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_9_0_X_2017-01-23-1100/external/slc6_amd64_gcc530/lib/libCore.so: file not recognized: File truncated
collect2: error: ld returned 1 exit status
gmake: *** [tmp/slc6_amd64_gcc530/src/SimMuon/RPCDigitizer/test/RPCEventDump/libRPCEventDump.so] Error 1
Leaving library rule at src/SimMuon/RPCDigitizer/test
>> Building edm plugin tmp/slc6_amd64_gcc530/src/SimMuon/RPCDigitizer/test/RPCFakeEvent/libRPCFakeEvent.so
>> Building edm plugin tmp/slc6_amd64_gcc530/src/SimMuon/RPCDigitizer/test/RPCDigiReader/libRPCDigiReader.so
/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_9_0_X_2017-01-23-1100/external/slc6_amd64_gcc530/lib/libCore.so: file not recognized: File truncated
</pre>

--- a/SimMuon/RPCDigitizer/python/muonRPCDigis_cfi.py
+++ b/SimMuon/RPCDigitizer/python/muonRPCDigis_cfi.py
@@ -78,7 +78,7 @@ if phase2_muon.isChosen():
             timeJitter = cms.double(1.0),
             IRPC_time_resolution = cms.double(0.1),
             IRPC_electronics_jitter = cms.double(0.025),
-           sigmaY = cms.double(1.), #math.sqrt((0.025)*(0.025)+(0.025)*(0.025))*0.66*299792458*1e+2*1e-9 = 0.7
+            sigmaY = cms.double(1.), #math.sqrt((0.025)*(0.025)+(0.025)*(0.025))*0.66*299792458*1e+2*1e-9 = 0.7
             do_Y_coordinate = cms.bool(True)
             ),       
                                      digiIRPCModel = cms.string('RPCSimModelTiming')

--- a/SimMuon/RPCDigitizer/src/RPCSimModelTiming.cc
+++ b/SimMuon/RPCDigitizer/src/RPCSimModelTiming.cc
@@ -176,7 +176,7 @@ void RPCSimModelTiming::simulate(const RPCRoll* roll,
           if(do_Y)
           {
 		adigi.hasY(true);
-                adigi.setY(smearedPositionY);
+		adigi.setY(smearedPositionY);
 		adigi.setDeltaY(sigmaY);
           }
           irpc_digis.insert(adigi);

--- a/SimMuon/RPCDigitizer/src/RPCSimModelTiming.cc
+++ b/SimMuon/RPCDigitizer/src/RPCSimModelTiming.cc
@@ -196,7 +196,7 @@ RPCDetId rpcId = roll->id();
   std::vector<float> veff = (getRPCSimSetUp())->getEff(rpcId.rawId());
   unsigned int nstrips = roll->nstrips();
   double area = 0.0;
-float striplength, xmin,xmax ;
+  float striplength, xmin,xmax ;
   if ( rpcId.region() == 0 )
     {
       const RectangularStripTopology* top_ = dynamic_cast<const

